instrumentation_sql.go

```diff
@@ -51,13 +53,26 @@ func InstrumentSQLDriver(sensor *Sensor, name string, driver driver.Driver) {
 // SQLOpen is a convenience wrapper for `sql.Open()` to use the instrumented version
 // of a driver previosly registered using `instana.InstrumentSQLDriver()`
 func SQLOpen(driverName, dataSourceName string) (*sql.DB, error) {
+
 	if !strings.HasSuffix(driverName, "_with_instana") {
 		driverName += "_with_instana"
 	}
 
 	return sql.Open(driverName, dataSourceName)
 }
 
+//go:linkname drivers database/sql.drivers
+var drivers map[string]driver.Driver
+
+// SQLInstrumentAndOpen combines `InstrumentSQLDriver` and `SQLOpen` calls
```

Comment:
I think adding some more information to the description makes sense 👍. Because, as you mentioned, this is not exactly the same as using `InstrumentSQLDriver ` and `SQLOpen`, here we reuse the "original" driver.